src/ucs/datastruct/ptr_array.h

```diff
@@ -73,9 +94,10 @@ void ucs_ptr_array_cleanup(ucs_ptr_array_t *ptr_array);
 /**
  * Insert a pointer to the array.
  *
- * @param value        Pointer to insert. Must be 8-byte aligned.
- * @param placeholder  Filled with placeholder value.
- * @return             The index to which the value was inserted.
+ * @param [in] ptr_array     Pointer to a ptr array.
+ * @param [in] value         Pointer to insert. Must be 8-byte aligned.
+ * @param [out] placeholder  Filled with placeholder value.
+ * @return                   The index to which the value was inserted.
```

Comment:
pls fix the `@return` section as suggested below

Reply:
?
I saw the return needs to be separated by a space line in string.h coding conversion - So I added a new line before each @return.